modules/__tests__/Link-test.js

```diff
@@ -557,4 +557,48 @@ describe('A <Link>', () => {
       })
     })
   })
+
+  describe('when the "noHref" prop is specified', function () {
+    class App extends Component {
+      render() {
+        return (
+          <div>
+            <Link to="/hello?the=query#hash" noHref>Link with noHref</Link>
+          </div>
+        )
+      }
+    }
+
+    it('returns an anchor tag without an href', function (done) {
+      render((
+        <Router history={createHistory('/')}>
+          <Route path="/" component={App} />
+        </Router>
+      ), node, function () {
```

Comment:
Oh, I see both are used in the file. At least be consistent in your own additions then (you use `() => {` below).